cmd/pd-server/main.go

```diff
@@ -22,6 +22,7 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/grpc-ecosystem/go-grpc-prometheus"
 	"github.com/juju/errors"
+	_ "github.com/pingcap/check"
```

Comment:
we need check package in vendor for CI test. but now glide update will remove it automatically. add this to keep it.